variables.tf

```diff
@@ -151,6 +151,12 @@ variable "runners_iam_instance_profile_name" {
   default     = ""
 }
 
+variable "runners_environment" {
+  description = "Environment variables in the pipeline, will be used in the runner config.toml"
```

Comment:
Could be helping to add here a sample in the description, such as: For example `["VAR=VALUE"]`

Reply:
I would like to do that, but I run into trouble:
If I put a hcl list into the description that's a syntax error, then if I escape it it's not anymore but it will look wrong in the generated docs :(
```hcl
variable "runners_environment_vars" {
  description = "Environment variables during build execution, e.g. `[\"KEY=Value\",\"TOKEN=xyz\"]`, see runner-public example. Will be used in the runner config.toml"
  type        = "list"
  default     = []
}
```
Any advice?